### PR TITLE
Use shared configs

### LINKED ISSUE #24: was-registrar: migrate to settings.yml / rails config / shared_config configuration framework
We need to setup the `config/settings.yml` configuration framework and migrate constants out of the Rails app configuration.

This issue is connected to https://github.com/sul-dlss/was-registrar/pull/22

### PR DESCRIPTION
**Pull request by ndushay**
#explicitly for shared_configs:

- changes to config files
- changes in files to use Settings.blah instead of Rails.configuration.blah
- updating gems
- changes to deploy to update the configs

Additional changes
- fix bug with bootstrap not finding expected font
- don't put public/assets in git
- some spec cleanup
- some deploy file cleanup
- avoid deprecation warning about config.serve_static_files
- update rubocop-todo

connected to #24

### CODE REVIEW

**Comment by drh-stanford** [comment]
Just to clarify, this PR is a refactoring (i.e.,  no functionality changes) for the configuration, right?

**Comment by ndushay** [comment]
Yes, the intent here is to switch to shared_configs and not change anything about the functionality.

A lot of refactoring crept in, as it tends to (for me).

File: config/deploy.rb [hunk 1]
```diff
@@ -1,13 +1,19 @@
 set :application, 'was-registrar'
 set :repo_url, 'https://github.com/sul-dlss/was-registrar.git'
+set :deploy_host, "was-registrar-#{fetch(:stage)}.stanford.edu"
+set :user, 'was'
 
 set :stages, %W(stage development production)
```
Comment (drh-stanford) [inline config/deploy.rb]: shouldn't this be dev stage prod? i'm not sure what `:stages` is used for though...

Comment (ndushay) [inline config/deploy.rb]: hmm - seems to work, but it's probably a typo ...  I will fix.

Comment (ndushay) [inline config/deploy.rb]: now I'm thinking it's never used ... will experiment

Comment (ndushay) [inline config/deploy.rb]: ha!  "ndushay committed on Jun 22, 2016"  -- old cruft that I introduced.

**Pull request merged**